modules/installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -19,7 +20,7 @@ For more information about creating a project in {ocp-short}, see link:https://d
 . From the *Developer Catalog* panel, click *Operator Backed*.
 . In the *Filter by keyword* box, enter _{product-short}_ and click the *{product}* card.
 . Click *Create*.
-. Add <<Adding a custom application configuration file to {ocp-short} using the Operator, custom configurations>> for the {product} instance.
```

Comment:
Are cross-references being replaced here? If so, please add the correct form of cross-references.

Reply:
This procedure, and this step in particular, needs to closer look, per RHIDP-2503